internal/pkg/vm/vm_test.go

```diff
@@ -1,140 +0,0 @@
-package vm
-
-import (
-	"context"
-	"testing"
-
-	"github.com/ipfs/go-cid"
-	"github.com/ipfs/go-datastore"
-	"github.com/ipfs/go-ipfs-blockstore"
-	xerrors "github.com/pkg/errors"
-
-	"github.com/filecoin-project/go-filecoin/internal/pkg/types"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/vm/actor"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/vm/errors"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/vm/state"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/vm2/vminternal/dispatch"
-
-	tf "github.com/filecoin-project/go-filecoin/internal/pkg/testhelpers/testflags"
-	"github.com/stretchr/testify/assert"
-)
-
-func TestTransfer(t *testing.T) {
```

Comment:
BLOCKING: It appears that `TestSendErrorHandling` made it `vmcontext_test`, but this test seems to have been lost. Please add it back or justify its exclusion.

Reply:
test is there, just different file. 
Moved it to vmcontext tests.